.travis.yml

```diff
@@ -9,5 +9,6 @@ before_install:
   - ./bazel-0.8.1-installer-linux-x86_64.sh --user
 
 script:
-  - bazel build -k //...
-  - bazel test -k //...
+  - bazel build --noshow_progress -k //...
+  - bazel test --noshow_progress -k //...
+  - ./format.sh
```

Comment:
I think travis should just verify that is well formatted. travis cannot change the code correct?

Reply:
Right. Travis doesn't make a commit. format.sh uses clang-format in place, reports which files were modified, and returns exitcode 1 for Travis.